### ISSUE TITLE
Design for UI Patterns Page

### ISSUE THREAD

**Issue by britanyponvelle** [IBM-Watson/design-library#163]

**Comment by britanyponvelle** [comment]
![wdl-ui-small-20](https://cloud.githubusercontent.com/assets/9538528/7091050/b7c25f76-df6c-11e4-99e0-4b523f73d531.png)

**Comment by britanyponvelle** [comment]
@kathrynmcelroy @bobbinrobyn  is this section structured like the other sections? Would the title here be "Buttons" instead of "Elements"?

**Comment by kathrynmcelroy** [comment]
This section will be different. There needs to be an introduction to what UI Patterns are (which is why I mentioned potentially reusing the pattern we create on the Home page), in addition to an intro paragraph at the beginning of the Elements, Components, and Layouts sections to explain what those terms mean.

We will not have the subsections in the nav until we determine how we want to sub-organize the elements/components/layouts. That way we don't have an ridiculous amount of nav links before you get to the content. That sub-organization will need to be proposed by @Snugug's experience and then user tested for confirmation.

**Comment by britanyponvelle** [comment]
Ok, I will put this as "blocked" until we sort that out and focus my attention on getting the markdown files for the other pages together.

**Comment by bobbinrobyn** [comment]
@everyone: I'm going to work on writing this content unless someone else has already done it/wants to do it.

>There needs to be an introduction to what UI Patterns are (which is why I mentioned potentially reusing the pattern we create on the Home page), in addition to an intro paragraph at the beginning of the Elements, Components, and Layouts sections to explain what those terms mean.

**Comment by bobbinrobyn** [comment]
Here's where I got to:

UI Patterns

This is a collection of customizable, reusable code and documentation that can be used to prototype cognitive applications and build them consistently. It is a starting point of common elements, components, and layouts that can and should be customized; it is not meant to be used as a definitive prescription for how Watson applications should be built. The UI patterns displayed here adhere to modern web standards and the principles and guidelines in the Watson Design Guide.


Elements

An element is the building block of a user interface. A combination of individual elements is used to create larger components and layouts. Buttons and form elements are examples of elements.

**Comment by bobbinrobyn** [comment]
@everyone: I'd love feedback on this content.

UI Patterns

This is a collection of customizable, reusable code and documentation that can be used to prototype cognitive applications and build them consistently. It is a starting point of common elements, components, and layouts that can and should be customized; it is not meant to be used as a definitive prescription for how Watson applications are to be built. The UI patterns displayed here adhere to modern web standards and the principles and guidelines in the Watson Design Guide.


Elements

An element is the building block of a user interface. Buttons and form elements are examples of individual elements. A combination of elements is used to create larger components and layouts.


Components

A component is a combination of elements. Navigation and search are examples of components. A combination of elements and components is used to create larger layouts.


Layouts

A layout is a combination of elements and components and provides the structure for an interface. Layouts are responsible for the sizing and positioning of elements and components on all or part of a page. Headers and footers may be examples of layouts.

**Comment by britanyponvelle** [comment]
Blocked by #136

**Comment by bobbinrobyn** [comment]
For the majority of the patterns, rendered previews should be the full width of the content area. Non-full width previews (for elements like buttons) will be the exception, not the default.

**Comment by britanyponvelle** [comment]
This issue has been broken into several smaller issues:

- [ ] Getting Started #184 
- [ ] Settings #185 
- [ ] Animation (docs) page #186 
- [ ] Javascript Standards page #187 
- [ ] Elements #188 
- [ ] Components #189 
- [ ] Layouts #190

**Comment by Snugug** [comment]
I wouldn't consider the Getting Started, Settings, or JavaScript standards to be part or the UI Patterns rendering but rather part of documentation. Should we go ahead and close this?

**Comment by britanyponvelle** [comment]
So would this section begin on Patterns> Elements ?

**Comment by Snugug** [comment]
@britanyponvelle This is a bit out of date. We have documentation, so the two main subsections are **Documentation** and **Patterns**

**Comment by britanyponvelle** [comment]
ok! On which page would the documentation live?

**Comment by britanyponvelle** [comment]
Nevermind! I see that it starts with Animation Documentation. 

:+1: 
I think we can close this

**Comment by bobbinrobyn** [comment]
:+1:

**Comment by kathrynmcelroy** [comment]
:+1:

**Comment by britanyponvelle** [comment]
One more :+1: and I think we can close this? This issue was meant to address the pages we need under UI patterns, which has been addressed if we are not including:

- [ ] Getting Started #184 
- [ ] Settings #185 
- [ ] Javascript Standards page #187 

In which case we can also close those pages

**Comment by poofichu** [comment]
:+1:

**Comment by poofichu** [comment]
Closing this.  We have this broken down into smaller issues.
